Jenkinsfile.yml

```diff
@@ -14,4 +20,99 @@ node('ubuntu18.04-OnDemand'){
     sh 'tar -cvzf demo.tar.gz demo'
     archiveArtifacts artifacts: 'demo.tar.gz', fingerprint: true, allowEmptyArchive: false
   }
-}
+    
+  try{
+    stage('Run SmokeTest'){
+      checkout([$class: 'GitSCM', branches: [[name: '*/master']], extensions: [[$class: 'RelativeTargetDirectory', relativeTargetDir: 'sdo-test']], userRemoteConfigs: [[credentialsId: 'sdo-automationgithubtoken', url: 'https://github.com/secure-device-onboard-ci/sdo-test']]])
+      sh '''
+        mkdir -p sdo-test/binaries/pri
+        mkdir -p sdo-test/binaries/supply-chain-tools
+        mkdir -p sdo-test/binaries/rendezvous-service
+        mkdir -p sdo-test/binaries/iot-platform-sdk
+        mkdir -p sdo-test/binaries/client-sdk/x86_ecdsa256_sct_bin
+        mkdir -p sdo-test/binaries/client-sdk/x86_ecdsa384_sct_bin
+      '''
+      print "copying the dependent binaries from pri, supply-chain-tools and rendezvous-service"
+      copyArtifacts filter: 'demo.tar.gz', fingerprintArtifacts: true, projectName: 'pri/master', selector: lastSuccessful(), target: 'sdo-test/binaries/pri/'
+      copyArtifacts filter: 'demo.tar.gz', fingerprintArtifacts: true, projectName: 'iot-platform-sdk/master', selector: lastSuccessful(), target: 'sdo-test/binaries/iot-platform-sdk/'
+      copyArtifacts filter: 'demo.tar.gz', fingerprintArtifacts: true, projectName: 'rendezvous-service/master', selector: lastSuccessful(), target: 'sdo-test/binaries/rendezvous-service'
+      copyArtifacts excludes: 'SmokeTest_Log.tar.gz', fingerprintArtifacts: true, projectName: 'client-sdk/master', selector: lastSuccessful(), target: 'sdo-test/binaries/'
+      print "Extract the dependent binaries to respective folders"
+      sh '''
+        tar -xvzf sdo-test/binaries/pri/demo.tar.gz -C sdo-test/binaries/pri/ --strip=1
+        tar -xvzf sdo-test/binaries/iot-platform-sdk/demo.tar.gz -C sdo-test/binaries/iot-platform-sdk/ --strip=1
+        tar -xvzf sdo-test/binaries/rendezvous-service/demo.tar.gz -C sdo-test/binaries/rendezvous-service/ --strip=1
+      '''
+      print "Copy client-sdk artifacts to sdo-test/binaries folder"
+      sh '''
+        tar -xvzf sdo-test/binaries/client-sdk/x86_ecdsa256_c_sct_bin.tar.gz -C sdo-test/binaries/client-sdk/x86_ecdsa256_sct_bin/ --strip=1
+        tar -xvzf sdo-test/binaries/client-sdk/x86_ecdsa384_c_sct_bin.tar.gz -C sdo-test/binaries/client-sdk/x86_ecdsa384_sct_bin/ --strip=1
+        cp -r sdo-test/binaries/client-sdk sdo-test
+        cp -r sdo-test/binaries/iot-platform-sdk sdo-test
+      '''
+      print "Copy SCT artifacts to sdo-test/binaries folder"
+      sh '''
+        cp -r demo/* sdo-test/binaries/supply-chain-tools/
+        mv sdo-test/binaries/iot-platform-sdk/ocs/ocsfs*.war sdo-test/binaries/iot-platform-sdk/ocs/ocsfs-.war
```

Comment:
This file rename might not be required as we are not running the WAR files individually. The existing docker file is able to pick the files in current naming format.